### DISCUSSION TITLE
Chrome Stable!

### DISCUSSION THREAD

**Discussion by joelgriffith** [browserless/chrome#3859] [Announcements]
We're happy to announce the availability of Chrome Stable to our docker images! Once this PR is merged: https://github.com/browserless/browserless/pull/3857, `ghcr.io/browserless/chrome` will _actually be_ Chrome and not Chromium. If you're intending to use Chromium please now use `ghcr.io/browserless/chromium`.

More details will be captured on our Changelog here: https://github.com/browserless/browserless/blob/main/CHANGELOG.md

**Comment by masterwishx** [comment]
What is the difference between ?

**Reply by joelgriffith** [reply]
There's quite a bit, but one is full-blown Chrome with all the trimmings, and the other is Chromium

**Reply by masterwishx** [reply]
But sujesstion was to move to chromium, moved already
